projects/js-packages/idc/components/idc-screen/screen-main.jsx

```diff
@@ -65,6 +66,34 @@ const ScreenMain = props => {
 				) }
 			</p>
 
+			{ dynamicSiteUrlDetected && (
+				<p>
+					{ createInterpolateElement(
+						customContent.dynamicSiteUrlText ||
+							__(
+								"<strong>Notice:</strong> It appears that your 'wp-config.php' file is using dynamic 'WP_SITEURL' or 'WP_HOME' values. " +
+									'Using dynamic values could cause Jetpack to enter Safe Mode, and might harm SEO. ' +
+									'<dynamicSiteUrlSupportLink>Learn how to set a static site URL instead.</dynamicSiteUrlSupportLink>',
```

Comment:
I think we need to ask JPOP Quill to take a look at the content.
They might think that the content is too technical, and we need to move the technical details into the linked support page.

Also, we probably need to either create a separate "dynamic URL" help page, or add a section to the existing page, and link it using the URI fragment (e.g. `#something`).

Reply:
Yeah, I was thinking about a new section no the https://jetpack.com/support/safe-mode/ page, and creating a new redirect link to that section. I'll swing back around to this once I've reached out to JPOP 👍